# HG changeset patch
# User Markus Mützel <markus.muetzel@gmx.de>
# Date 1638187375 -3600
#      Mon Nov 29 13:02:55 2021 +0100
# Node ID 3d28b7df6df9c5c161f9e4518a03c34d6300f2b4
# Parent  b38448d3d291256bedb125b2c52c40820acdc235
Fix compilation with Octave 8 (bug #61582).

* src/error-helpers.h: Use wrapper function.

diff -r b38448d3d291 -r 3d28b7df6df9 src/error-helpers.h
--- a/src/error-helpers.h	Wed Feb 17 13:25:40 2021 +0100
+++ b/src/error-helpers.h	Mon Nov 29 13:02:55 2021 +0100
@@ -64,7 +64,7 @@
       } \
     catch (OCTAVE__EXECUTION_EXCEPTION& e) \
       { \
-        verror (e, __VA_ARGS__); \
+        c_verror (e, __VA_ARGS__); \
       }
 #endif
 
